debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -255,18 +251,29 @@ private Schema getValueSchema(Schema debeziumEventSchema, Integer version) {
     private SchemaBuilder getSchemaBuilder(Schema debeziumEventSchema) {
         SchemaBuilder schemaBuilder = SchemaBuilder.struct();
 
+        // eventType is first from due to backward compatibility reasons
```

Comment:
Didn't we conclude that sole re-ordering isn't a problem in terms of backwards-compatibility? That said, eventType used to come before payload, whereas now it doesn't any longer, so isn't there an ordering change in any case?

Reply:
@gunnarmorling Yes we did but it felt to me this is cheap price to keep it same.

Reply:
@jpechane, but the order still *is* changed, no? So it seems the current approach adds some complexity for not much gain.

Reply:
@gunnarmorling No with current approach the order is kept

Reply:
How so? It used to come *before* the payload, and now it's *after* that, no?

Reply:
@gunnarmorling No, it first finds eventType, then adds payload and then it add the rest of additional fields

Reply:
Ah yes, I stand corrected. But taking a step back -- the idea was to *simplify* things and make them more regular, I don't feel this is what's happening here ;) Also keep in mind the feature is marked as "incubating" in the docs, so I don't think we should bother too much for keeping that ordering here, esp. as it doesn't have any implications on compatibility anyways.

Another question: it seems there's no way to *not* have the event type field in the messages, or is there? Tbh. I think I'd just remove any notion of "event type" from the implementation altogether. After all, it's not an inherent detail of the pattern (unlike the aggregate type which definitely is needed for topic routing). So if the user wants to have it, they can just add it themselves as an additional field.